guides/json.md

```diff
@@ -0,0 +1,66 @@
+- [Making Raw JSON REST Requests](#making-raw-json-rest-requests)
+  - [GET](#get)
+  - [PUT](#put)
+  - [POST](#post)
+  - [DELETE](#delete)
+
+# Making Raw JSON REST Requests
+
+The OpenSearch client implements many high-level REST DSLs that invoke OpenSearch APIs. However you may find yourself in a situation that requires you to invoke an API that is not supported by the client. Use `client.transport.perform_request` to do so. See [samples/json](samples/json) for a complete working sample.
```

Comment:
Link "samples/json" is not working because it is considered as "opensearch-py/guides/samples/json". We need "opensearch-py/samples/json"